content/docs/instrumenting/exporters.md

```diff
@@ -139,6 +139,7 @@ practices.
    * Clojure: [prometheus-clj](https://github.com/soundcloud/prometheus-clj)
    * Go: [go-metrics instrumentation library](https://github.com/armon/go-metrics)
    * Go: [gokit](https://github.com/peterbourgon/gokit)
+   * Go: [prombolt](https://github.com/mdlayher/prombolt)
```

Comment:
This should go to the Databases section I believe, as it is an exporter for bolt?